src/Microsoft.IdentityModel.Tokens/CallContext.cs

```diff
@@ -55,6 +55,14 @@ public CallContext(Guid activityId)
             ActivityId = activityId;
         }
 
+        /// <summary>
+        /// Creates a default context that doesn't capture any logs.
+        /// </summary>
+        public static CallContext Default
```

Comment:
I don't think this buys us much.
Same as:
`new CallContext();`

Reply:
Agreed. I'll remove it.

Reply:
Addressed with https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1260/commits/a7c7b608fbd0da084e479c531b45120cf8f738c6